tests/checks/check-namespace-labels.yaml

```diff
@@ -0,0 +1,19 @@
+---
+- name: "{{ testname }} - Check if {{project}} namespace is labelled"
```

Comment:
nice! definitely wanted to implement a test to check something via kubectl 💚

Reply:
:+1: